src/components/data-processing/ConfigureEmbedding/ConfigureEmbedding.jsx

```diff
@@ -7,38 +7,33 @@ import {
 } from 'antd';
 
 import { isUnisample } from 'utils/experimentPredicates';
-import CalculationConfig from './CalculationConfig';
-import MiniPlot from '../../plots/MiniPlot';
+import MiniPlot from 'components/plots/MiniPlot';
 
-import CategoricalEmbeddingPlot from '../../plots/CategoricalEmbeddingPlot';
-import ContinuousEmbeddingPlot from '../../plots/ContinuousEmbeddingPlot';
+import CategoricalEmbeddingPlot from 'components/plots/CategoricalEmbeddingPlot';
+import ContinuousEmbeddingPlot from 'components/plots/ContinuousEmbeddingPlot';
 
 import {
   updatePlotConfig,
   loadPlotConfig,
   savePlotConfig,
-} from '../../../redux/actions/componentConfig';
+} from 'redux/actions/componentConfig';
 
-import PlotStyling from '../../plots/styling/PlotStyling';
-import { updateFilterSettings } from '../../../redux/actions/experimentSettings';
-import loadCellMeta from '../../../redux/actions/cellMeta';
-import generateDataProcessingPlotUuid from '../../../utils/generateDataProcessingPlotUuid';
-import Loader from '../../Loader';
-import { getCellSets } from '../../../redux/selectors';
+import PlotStyling from 'components/plots/styling/PlotStyling';
+import loadCellMeta from 'redux/actions/cellMeta';
+import generateDataProcessingPlotUuid from 'utils/generateDataProcessingPlotUuid';
+import Loader from 'components/Loader';
+import { getCellSets } from 'redux/selectors';
+import CalculationConfig from 'components/data-processing/ConfigureEmbedding/CalculationConfig';
 
 const { Panel } = Collapse;
 
 const ConfigureEmbedding = (props) => {
   const { experimentId, onConfigChange } = props;
   const [plot, setPlot] = useState(null);
+  const filterName = 'configureEmbedding';
```

Comment:
I don't see where you use this `filterName` in this file. Can you point me a link to where it's used?

Reply:
This is used on line 56 in this file

Reply:
yup makes sense, I didn't see the deleted line for `filterName`